banks.js

```diff
@@ -20,5 +20,6 @@ module.exports = [
   { name: 'UNITED BANK FOR AFRICA PLC', code: 'UBA' },
   { name: 'UNITY BANK PLC', code: 'UNB' },
   { name: 'WEMA BANK PLC', code: 'WEM' },
-  { name: 'ZENITH BANK PLC', code: 'ZIB' }
+  { name: 'ZENITH BANK PLC', code: 'ZIB' },
+  { name: 'JAIZ BANK PLC', code: 'JAIZ' }
```

Comment:
So I'm going to let this go in but recently I discovered there's a uniform code banks in nigeria share and it's numerical.

So at the end the code will be changed to actual code. If you want to work on it, it'd be nice but it's not compulsory (let me know either way if you will or won't).

If you check this [link](https://developers.paystack.co/v1.0/reference) paystack exposes an endpoint that can be used to get this code.

What I'm proposing is the structure of the object becomes:
```js
{ name: 'Bolaji Bank', code: 203, slug: 'BBK' }
```

Reply:
Sure, I'll be willing to be involved!